stock_package_nbr/models/stock_picking.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2017 Sylvain Van Hoof (Okia) <sylvain@okia.be>
+# Copyright 2018-2019 Jacques-Etienne Baudoux (BCIM) <je@bcim.be>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+from odoo.tools import float_compare
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    def put_in_pack(self):
+        self.ensure_one()
+        if not self.move_line_ids.filtered(
```

Comment:
This condition is very hard to read.... IMO it would be nice to add a comment to explain the process into this override of put_in_pack.